babybuddy/settings/docker.py

```diff
@@ -4,13 +4,14 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
+# Load settings from env file / variables with fallback defaults to support current psql deployment
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
We also want this to be configurable, so we should use e.g. `DB_ENGINE`.

And similarly, lets use the `DB_` prefix for all of these settings --

- `DB_ENGINE`
- `DB_NAME` (for `POSTGRES_DB`)
- `DB_USER`
- `DB_HOST`
- `DB_PORT`

Reply:
Makes a ton of sense and I'm making the edits now. One question I had was about your plan on handling the rollout for additional database types and how best to lay the foundation for that moving forward. I've seen examples of people checking the db_engine variable and wrapping the proper database structure within an if/else statement right inside of the settings.py file, and I've seen people break out the different DB structures into other files like settings.psql.py that would supercede the original settings. I'm assuming the latter would require the app to check the env and call the proper settings file somewhere else (or put that on the user to issue as a command when creating their docker container) whereas the former could theoretically handle the adjustments without having to make additional code changes.

Would you have a preference on how you want to proceed?

Reply:
Changes pushed in 0c05cd3 and 6f83e16, discussion about above question took place in the other conversation